src/bsp/src/stm32/include/UWBMessages.h

```diff
@@ -34,11 +34,17 @@ namespace UWBMessages {
         FunctionCode m_functionCode;
     };
 
+    struct TWRPoll {
+        DWFrame m_header_frame;
+    };
+
     struct TWRResponse {
+        DWFrame m_header_frame;
         uint64_t m_calculatedTOF;
```

Comment:
Is the calculated TOF still used? If not, remove it to save airtime

Reply:
let for futur use, see https://www.google.com/search?channel=fs&client=ubuntu&q=DecaRanging+ARM+Source+Code+Guide